### PR TITLE
Py3 in audit event

### PR DESCRIPTION
**Pull request by dPeS**
tox checked on py27, py35 (don't have 34)

### CODE REVIEW

File: rwslib/extras/audit_event/test_context.py [hunk 1]
```diff
@@ -9,4 +9,4 @@ def setUp(self):
         pass
 
     def test_repr(self):
-        ContextBase().__repr__()
+        assert ContextBase().__repr__(), ''
```
Comment (glow-mdsol) [inline rwslib/extras/audit_event/test_context.py]: `self.assertEqual`?

File: rwslib/extras/audit_event/main.py [hunk 2]
```diff
@@ -1,10 +1,14 @@
 # -*- coding: utf-8 -*-
 __author__ = 'isparks'
 
-from rwslib import RWSConnection
 from rwslib.rws_requests.odm_adapter import AuditRecordsRequest
-from urlparse import urlparse, parse_qs
-from parser import parse
+
+try:
```
Comment (glow-mdsol) [inline rwslib/extras/audit_event/main.py]: Any advantage over using `six` here?  Also, maybe `ImportError` rather than a generic except?

**Comment by dPeS** [comment]
it's a pytest way thought it is used, I was wrong -> changing

File: rwslib/extras/audit_event/main.py [hunk 3]
```diff
@@ -11,6 +11,12 @@
 from rwslib.extras.audit_event.parser import parse
 import logging
 
+# Python 3
```
Comment (isparks) [inline rwslib/extras/audit_event/main.py]: I think we can stick with int everywhere here. In Python2.7:

Python 2.7.11 (default, Jan 22 2016, 08:29:18) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> type(int(999999999999999999999999999999999999999999999999999999999999))
<type 'long'>
>>> 

So python 2.7 auto converts an int() to a Long if required. This is also mentioned in 2.7 docs but I didn't know this before looking into it.

File: rwslib/extras/audit_event/main.py (hunk 2 above)
Comment (dPeS) [inline rwslib/extras/audit_event/main.py]: six don't support that, will do ImportError

File: rwslib/extras/audit_event/test_context.py (hunk 1 above)
Comment (dPeS) [inline rwslib/extras/audit_event/test_context.py]: fixed

File: rwslib/extras/audit_event/main.py (hunk 3 above)
Comment (dPeS) [inline rwslib/extras/audit_event/main.py]: sure, didn't went that deep -> changing

File: rwslib/extras/audit_event/main.py (hunk 2 above)
Comment (glow-mdsol) [inline rwslib/extras/audit_event/main.py]: see `test_biostats_gateway.py`

```python
from six.moves.urllib.parse import urlparse, parse_qs
```

Comment (dPeS) [inline rwslib/extras/audit_event/main.py]: damn, info was based on: http://python3porting.com/stdlib.html#urllib-urllib2-and-urlparse

changing :)

Comment (dPeS) [inline rwslib/extras/audit_event/main.py]: done

**Comment by dPeS** [comment]
I've changed parse_long -> parse_int, howto regenerate sphinx?

File: rwslib/extras/audit_event/test_context.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+import unittest
+from rwslib.extras.audit_event.context import ContextBase
+
+
+class ContextBaseTaseCase(unittest.TestCase):
```
Comment (anewbigging) [inline rwslib/extras/audit_event/test_context.py]: Should this class be called 'ContextBaseTestCase'?

File: rwslib/extras/audit_event/test_odmadapter.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        self.count = 0
+
+    def NoMatchHere(self, context):
+        self.count += 1
+
+    def default(self, context):
+        pass
+
+
+class OdmAdapterTaseCase(unittest.TestCase):
```
Comment (anewbigging) [inline rwslib/extras/audit_event/test_odmadapter.py]: Should this class be called 'ODMAdapterTestCase'?

**Comment by isparks** [comment]
@dPeS Sphinx docs are regenerated on a merge to master. If you want to contribute to the docs (please do!) then just edit the .rst files in the docs directory. make html from there will make the documentation locally if you want to check it (generated docs go in the build sub-directory which we don't check in since it's a product.

File: rwslib/extras/audit_event/test_odmadapter.py (hunk 5 above)
Comment (dPeS) [inline rwslib/extras/audit_event/test_odmadapter.py]: fixing

File: rwslib/extras/audit_event/test_context.py (hunk 4 above)
Comment (dPeS) [inline rwslib/extras/audit_event/test_context.py]: see no difference?

Comment (isparks) [inline rwslib/extras/audit_event/test_context.py]: ContextBaseT**a**s**e**Case
ContextBaseT**e**s**t**Case

**Comment by isparks** [comment]
OK. Looking good. Are we ready to merge?

**Comment by dPeS** [comment]
+1 for me

**Pull request merged**